torch/nn/qat/modules/linear.py

```diff
@@ -49,3 +50,11 @@ def from_float(cls, mod):
         qat_linear.weight = mod.weight
         qat_linear.bias = mod.bias
         return qat_linear
+
+    def to_float(self):
```

Comment:
given @HDCharles recent work, might be good to copy over the right device and `is_traning` setting to the new module

Reply:
I think we already take care of the device when we copy the weight? since it's used when we initialize thew weight: https://github.com/pytorch/pytorch/blob/master/torch/nn/modules/linear.py#L81

Reply:
will add is_training